src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/AttributeCollection.cs

```diff
@@ -59,7 +61,7 @@ protected AttributeCollection() : this(Array.Empty<Attribute>())
         /// <summary>
         /// Creates a new AttributeCollection from an existing AttributeCollection
         /// </summary>
-        public static AttributeCollection FromExisting(AttributeCollection existing, params Attribute[] newAttributes)
+        public static AttributeCollection FromExisting(AttributeCollection existing, params Attribute[]? newAttributes)
```

Comment:
It's strange to see a params argument be nullable.  Do we actually expect folks to pass in a null array here, even though technically the implementation allows it?  I'd prefer to just remove this '?'.